OpenTaiko/src/Common/C定数.cs

```diff
@@ -3,7 +3,7 @@
 namespace OpenTaiko {
 
 	/// <summary>
-	/// 難易度。
+	/// Difficulty degree.
```

Comment:
"Difficulty" is enough.

Reply:
I got rid of the doc here completly since it has no use if we just mimic the Enum name.